aries-site/src/examples/templates/FilterControls/FilterControls.js

```diff
@@ -41,19 +42,28 @@ export const FilterControls = ({
   }, [syncFilteredResults]);
 
   return (
-    <Box gap="xsmall" flex={false}>
-      <Box direction="row" align="end" gap="small">
-        {searchFilter && (
-          <SearchFilter placeholder={searchFilter.placeholder} />
-        )}
-        <Filters />
+    <Box flex={false} fill="horizontal">
+      <Box direction="row" align="start" justify="between" wrap>
+        <Box
+          direction="row"
+          align="start"
+          gap="small"
+          margin={{ bottom: 'xsmall' }}
```

Comment:
Should this Box have `flex`?

Reply:
I am no noticing any difference when having `flex` vs not having `flex.

Reply:
OK. Maybe that's a default.